docs/source/tutorials/configurable.rst

```diff
@@ -0,0 +1,108 @@
+==============================
+Configuration and plugin setup
+==============================
+
+Maybe you :doc:`played with commands </tutorials/playing-command>` for your
+plugin and now you want to make your plugin configurable. If you run an
+instance of Sopel yourself, you probably had to open and edit its configuration
+file.
+
+Usually located in your home directory, under the ``.sopel/`` folder, the
+configuration file is an INI file with sections defined by Sopel's core and by
+plugins. In this tutorial, let's see how to declare and use a configuration
+section dedicated to your plugin.
+
+
+Declaring your configuration
+============================
+
+To declare a configuration section, you must first create a subclass of
+:class:`~sopel.config.types.StaticSection`, and define attributes::
+
+    from sopel.config import types
+
+    class MyPluginSection(types.StaticSection):
+        fruits = types.ListAttribute('fruits')
+
+
+Telling Sopel about it
+======================
+
+Now, having a class in your plugin doesn't achieve much: you need to tell the
+bot about it. The best place to do so is in the :func:`setup` function
+hook of your plugin::
+
+    def setup(bot):
+        bot.settings.define_section('myplugin', MyPluginSection)
+
+This way, you tell Sopel that the ``[myplugin]`` section is used by your
+plugin, and to parse this section Sopel must use your class, i.e.
+``MyPluginSection``.
+
+
+Using your section
+==================
+
+In your configuration file, you can add the following lines:
```

Comment:
Slight reword to suggest more strongly that this new section means nothing to Sopel without the steps that came before.

```suggestion
Now that you have told Sopel about your custom section, you can add the
following lines in your configuration file:
```